app/views/mods_panel.py

```diff
@@ -720,7 +721,7 @@ def eventFilter(self, source_object: QObject, event: QEvent) -> bool:
                         mod_name = mod_metadata.get("name")
                         mod_folder_name = mod_metadata["folder"]
                         mod_folder_path = mod_metadata["path"]
-                        publishedfileid = mod_metadata.get("publishedfileid")
+                        publishedfileid: str = mod_metadata.get("publishedfileid")
```

Comment:
Technically, the typing here is a bit of a stopgap already since the true type that it should be is Any. I think based on the behavior down stream, it would be better if it was coded from scratch to be None, but for our case it would be better to be an empty string as fallback. 

Again, annoying consequences of storing the metadata as several nested dictionaries.